### ISSUE TITLE
Embed KRS Xilinx Adapt videos to KRS docs

### ISSUE THREAD

**Issue by vmayoral** [Xilinx/KRS#12]
@jasvinderkhurana could you please check and ask around if we're allowed to embed the KRS Adapt talks at https://xilinx.github.io/KRS/sphinx/build/html/docs/intro.html?

I think it'll add significant value to have those two talks available in the intro pages of the documentation. One easy way to do this is to upload them to YouTube and embed them in Markdown.

**Comment by jasvinderkhurana** [comment]
@vmayoral, Yes I think that is a great idea, I can check with team and try to get the adapt videos embed in documentation.
